app/src/main/java/org/fossasia/openevent/activities/BaseActivity.java

```diff
@@ -38,4 +42,18 @@ protected void onResume() {
         Timber.i("Activity onResume");
         super.onResume();
     }
+
+    public void helperHideKeyboard(View view) {
+        view.setOnTouchListener(new View.OnTouchListener() {
```

Comment:
I thought we weren't doing this? Because this setups a touch listener and hides a bit of critical functionality.